tool/src/main.rs

```diff
@@ -54,12 +58,24 @@ fn main() {
         tas = Tas::new().unwrap();
         println!("TAS created successfully.");
     }
-
-    let script_file = env::args().skip(1).next().unwrap_or("tas.lua".to_string());
+    let script_file = if env::args().len() == 1 {
+        if std::env::current_dir().unwrap().join("tas.lua").metadata().is_err() {
```

Comment:
If you use `Path::new`, just like `File::open`, it'll automatically start its path from the current directory. Therefore you don't need to actually first get the `current_dir`, but can directly use `Path::new`.
Also, there is a function [`Path::is_file`](https://doc.rust-lang.org/std/path/struct.Path.html#method.is_file).